lib/puppet/pops/adapters.rb

```diff
@@ -85,6 +85,13 @@ def self.loader_for_model_object(model, file = nil, default_loader = nil)
 
     class PathsAndNameCacheAdapter < Puppet::Pops::Adaptable::Adapter
       attr_accessor :cache, :paths
+
+      def self.create_adapter(env)
```

Comment:
For posterity, `create_adapter` is always called as part of [`adapt`](https://github.com/puppetlabs/puppet/blob/main/lib/puppet/pops/adaptable.rb#L98).